frontend-react/src/views/Home/index.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
-import { useTranslation } from 'react-i18next';
 
 import Page from '../../components/Page';
+import useT from "../../utils/translation";
 
 const LoginView = () => {
-  const { t } = useTranslation();
+  // I know this line sucks, but there's no better way to handle this long string now, unfortunately
```

Comment:
You can try template literal with backticks

Reply:
Nope, won't work. This stuff is rather complex, to be honest (I did investigate that to the core).

Basically `transifex` parses every `*.js` file that it finds using `babel`. It iterates trough the whole AST of every file and tries to find calls to `t()` or `useT()` (it assumes that these calls are dedicated to transifex. When it finds these calls (as in this exact line) it picks the first passed argument from the AST and checks if it's a string literal. If it is, then it collects it for pushing, but if it's anything else (even a templated literal with backticks) it fails!.

I'm working on a PR that will add support for more relaxed arguments (for example `useT("a" + "b")` does not work yet!), but I need a bit more time for that.

So until then, we're truly stuck with this approach.

Reply:
Not true anymore! We can replace it with concatenated const string as per this feature: https://github.com/transifex/transifex-javascript/pull/53